frontend/Auth/FinishResetPassword.tsx

```diff
@@ -171,7 +261,16 @@ export function FinishResetPassword(props: RouterProps) {
 }
 
 interface FinishResetPasswordFormData {
-  password: string;
+  password: string | undefined;
```

Comment:
It's more common to use the shorthand syntax for this, unless there's a nuanced behavior of `?:` that you're trying to avoid:

```suggestion
  password?: string;
```

Same thing applies to line 268, 272, 273

Reply:
Thank you!